cpu/kinetis/include/periph_cpu.h

```diff
@@ -130,9 +130,33 @@ typedef uint16_t gpio_t;
 #define PERIPH_TIMER_PROVIDES_SET
 
 /**
- * @brief   number of usable power modes
+ * @name    Kinetis power mode configuration
+ * @{
+ */
+#define PM_NUM_MODES    (4U)
+enum {
+    KINETIS_PM_LLS  = 0,
+    KINETIS_PM_VLPS = 1,
+    KINETIS_PM_STOP = 2,
+    KINETIS_PM_WAIT = 3,
+};
+#define PM_BLOCKER_INITIAL  0x01010101
+#if MODULE_PM_LAYERED
+#include "pm_layered.h"
+/**
+ * @brief   pm_block iff pm_layered is used
  */
-#define PM_NUM_MODES    (1U)
+#define PM_BLOCK(x) pm_block(x)
+/**
+ * @brief   pm_unblock iff pm_layered is used
+ */
+#define PM_UNBLOCK(x) pm_unblock(x)
+#else
+/* ignore these calls when not using pm_layered */
+#define PM_BLOCK(x)
+#define PM_UNBLOCK(x)
```

Comment:
drivers/kw41zrf/include/kw41zrf_intern.h redefines these, can you adapt that to use this defintions?

Reply:
In agree. But in fact I think we probably need to generalize it further even. Unless I missed something, everything that calls `pm_block()` and `pm_unblock()` is going to need some equivalent of these defines to cover the case where `pm_layered` isn't built. Is that wrong?

Reply:
I think you are right, currently `pm_%` is only used in code for platforms that always provide `pm_layered` or there are a couple of `ifdef MODULE_PM_LAYERED`, maybe it makes sense to have empty definitions of those functions, but I think this should be in its own PR though, might spark up some unneeded discussion (in the context of this PR)

Reply:
In the mean time I removed the duplication as you indicated.